.github/workflows/conformance.yml

```diff
@@ -33,12 +33,12 @@ jobs:
             api-gateway-image: "hashicorppreview/consul-api-gateway:0.4-dev"
             consul-image: "hashicorp/consul:1.11"
             envoy-image: "envoyproxy/envoy:v1.20-latest"
-            consul-k8s-version: "0.45.0"
+            consul-k8s-version: refs/tags/v0.45.0
```

Comment:
This will need to reference your branch of consul-k8s w/ the added ReferenceGrant permissions I think

Reply:
```suggestion
            consul-k8s-version: "charts/capigw-controller-clusterrole-referencegrants"
```